ape/healthz.py

```diff
@@ -0,0 +1,18 @@
+from flask import Blueprint
+from healthcheck import HealthCheck
+
+
+healthz_blueprint = Blueprint('healthz', __name__)
+
+
+def check_ape():
+    return True, "ape ok"
```

Comment:
It would be worth to try checking if the auth0 connection is doing all right. This would verify if all secrets are configured properly etc.

Reply:
If we want to check if configuration is ok, we can send one request when app is starting and save the result. Getting management API token every 20 s (or whatever the period for health checks is) just to send request would be a little too much.